next.config.js

```diff
@@ -62,17 +66,15 @@ module.exports = withPlausibleProxy()({
       {
         source: '/.well-known/:slug*',
         headers: [
-          ...corsHeaders
+          ...corsHeaders,
+          noCacheHeader
```

Comment:
I set no caching here since we include a `k1` in the response payload that's unique for each request.